server/container/cassandra-guice/src/main/java/org/apache/james/jmap/JMAPModule.java

```diff
@@ -52,6 +51,7 @@ JMAPConfiguration provideConfiguration(FileSystem fileSystem) throws Configurati
                 .keystore(configuration.getString("tls.keystoreURL"))
                 .secret(configuration.getString("tls.secret"))
                 .jwtPublicKeyPem(loadPublicKey(fileSystem, Optional.ofNullable(configuration.getString("jwt.publickeypem.url"))))
+                .port(Optional.ofNullable(configuration.getInteger("jmap.port", DEFAULT_JMAP_PORT)))
```

Comment:
I would use `Optional.of` method, i don't think we want to manage random port at runtime.